pan/Makefile

```diff
@@ -6,7 +6,7 @@ IMAGE_GLOB=*.qcow2
 # match versions like:
 # PA-VNM-KVM-7.0.1.qcow2
 # PA-VM-KVM-10.0.6.qcow2
-VERSION=$(shell echo $(IMAGE) | sed -e 's/.*-\([0-9]\{1,2\}\.[0-9]\{1,2\}.[0-9]\{1,2\}\)\.qcow2$$/\1/')
+VERSION=$(shell echo $(IMAGE) | sed -e 's/PA-\(VNM\|VM\)-KVM-\(.*\).qcow2/\2/')
```

Comment:
@vista- maybe for now until we figure out what image name scheme to adopt for all nodes, lets switch this one to the enhanced re?